index.html

```diff
@@ -19,22 +19,201 @@
+          </div>
+        </div>
+        <div class="achievement-detail">
+          <div class="achievement-detail_pic">
+            
+          </div>
+          <div class="achievement-detail_brief">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="teck-stack">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="the-button">See project</button>
+          </div>
+        </div>
+        <div class="achievement-detail">
+          <div class="achievement-detail_pic">
+            
+          </div>
+          <div class="achievement-detail_brief">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="teck-stack">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="the-button">See project</button>
+          </div>
+        </div>
+        <div class="achievement-detail">
+          <div class="achievement-detail_pic">
+            
+          </div>
+          <div class="achievement-detail_brief">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="teck-stack">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="the-button">See project</button>
+          </div>
+        </div>
+        <div class="achievement-detail">
+          <div class="achievement-detail_pic">
+            
+          </div>
+          <div class="achievement-detail_brief">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="teck-stack">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="the-button">See project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="my-personal-info">
+      <h2>About Me</h2>
+      <div class="background-extreme-corner"></div>
+      <p class="my-ability"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+      <button class="the-button">Get my resume</button>
+      <div class="background-extreme-bottom-corner"></div>
+      <div class="my-ability-display">
+        <ul class="my-ability-display-list">
+          <li>
+            <div class="skill-list">
+              <img src="./assets/images/languages.png" alt="Languages">
```

Comment:
- [ ] Please make sure that the images in the "technologies" section are loading as there are no images with this name in your "assets" directory.